python/Vertx/Eventbus.py

```diff
@@ -218,10 +218,10 @@ def publish(self, address, body, deliveryOption=None):
 
             if replyAddress == None:
```

Comment:
`replyAddress` is not used at all when `publish`ing the messages, would you please remove it ?

It would be even better if you can [sign-off your commit](https://github.com/eclipse-vertx/vert.x/blob/master/CONTRIBUTING.md#contributions), thanks :)

Reply:
actually now commits don't need to be signed anymore @gaol , I'll update the wiki page